components/molecule/accordion/src/index.scss

```diff
@@ -0,0 +1,41 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import '../../collapsible/src/index';
+@import './settings';
+.sui-MoleculeAccordion {
+  .sui-MoleculeCollapsible {
```

Comment:
Not sure if this a good practice. IMHO we shouldn't overwrite other component styles.
Is `MoleculeCollapsible` definition providing a borders specification? if so, maybe we can send a prop to the `MoleculeCollapsible` in order to handle there the style changes. Otherwise UX could  rethink `MoleculeCollapsible` specification. 
This discussion starts to be recurrent. More opinions about that would be apreciated.

Reply:
I do think as @JoanClaret, if we overwrite styles, the components are less consistent. I think that's an anti-pattern we could avoid. IMHO situations like this one should make us send a re-definition to UX or think about the limitations of the component

Reply:
It's not only the _border_ property that has to be changed,  all the design from `MoleculeCollapsible` doesn't fit with `MoleculeAccordion`'s design. The initial idea was to reuse the behaviour from `MoleculeCollapsible` as children for `MoleculeAccordion` but not the styles. So, would you see it better if this styles were in `MoleculeCollapsible`'s css as a special case when nested in `MoleculeAccordion`?

Reply:
ok, i see. 
Considering that `MoleculeCollapsible` has totally different styling, and its functionality does not need a large development (just toggle a layer), for me, the risks of overwriting classes and break the component isolation are bigger than the benefit of importing this component , so I would simply bet for develop `MoleculeAccordion` from scratch.

Reply:
@adriadoz please let's talk about this